docs/conf.py

```diff
@@ -30,19 +30,16 @@
 
 # General configuration -------------------------------------------------------
 
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.viewcode']
+extensions = [
+    'autoapi.extension',
+    'sphinx.ext.autodoc',
+    'sphinx.ext.viewcode'
+]
+autoapi_dirs = ['../tests', '../robottelo']
```

Comment:
Personally, I wouldn't mind if we removed `../tests` from that list.

Looks like we never generated it in the past (I mean, except for list of modules), and there is hardly anything usable there. I mean - what would I need to be doing so it would be easier for me to browse docs instead of just navigating the file? I can't really think of use-case for that.

Thoughts?

Reply:
I'm fine with removing this as well.